docs/API.md

```diff
@@ -45,6 +45,9 @@
     of objects. These functions should return an JSONifiable object and they
     should never throw. When logging an object, each top-level property matching the exact key of a serializer
     will be serialized using the defined serializer.
+    
+    **USE ON YOUR OWN RISK**
+    As an exception it is possible to register `Symbol.for('*')` serializer, which accepts the complete log object (`pid`, `hostname` and all custom properties) and returns the updated complete log object. 
```

Comment:
"Alternatively, it is possible to register a serializer under the key `Symbol.for('*')` which will act upon the complete log object, i.e. every property."

Reply:
updated